src/uct/api/uct_def.h

```diff
@@ -152,7 +174,7 @@ typedef struct uct_iov {
  *
  */
 typedef ucs_status_t (*uct_am_callback_t)(void *arg, void *data, size_t length,
-                                          void *desc);
+                                          unsigned flags);
```

Comment:
desc has been dropped from the argument list, yet it is still referenced in the function description.  It should be clearer as to how the callback gets the desc pointer.  In particular, the discussion of UCT_AM_FLAG_DESC is not well coordinated  or referenced in the detailed description of uct_am_callback_t.